third_party/doj/alphanum.hpp

```diff
@@ -2,38 +2,41 @@
-containing numbers.  Instead of sorting numbers in ASCII order like a
-standard sort, this algorithm sorts numbers in numeric order.
+   The Alphanum Algorithm is an improved sorting algorithm for strings
+   containing numbers.  Instead of sorting numbers in ASCII order like a
+   standard sort, this algorithm sorts numbers in numeric order.
 
-The Alphanum Algorithm is discussed at http://www.DaveKoelle.com
+   The Alphanum Algorithm is discussed at http://www.DaveKoelle.com
 
-This implementation is Copyright (c) 2008 Dirk Jagdmann <doj@cubic.org>.
-It is a cleanroom implementation of the algorithm and not derived by
-other's works. In contrast to the versions written by Dave Koelle this
-source code is distributed with the libpng/zlib license.
+   This implementation is Copyright (c) 2008 Dirk Jagdmann <doj@cubic.org>.
+   It is a cleanroom implementation of the algorithm and not derived by
+   other's works. In contrast to the versions written by Dave Koelle this
+   source code is distributed with the libpng/zlib license.
 
-This software is provided 'as-is', without any express or implied
-warranty. In no event will the authors be held liable for any damages
-arising from the use of this software.
+   This software is provided 'as-is', without any express or implied
+   warranty. In no event will the authors be held liable for any damages
+   arising from the use of this software.
 
-Permission is granted to anyone to use this software for any purpose,
-including commercial applications, and to alter it and redistribute it
-freely, subject to the following restrictions:
+   Permission is granted to anyone to use this software for any purpose,
+   including commercial applications, and to alter it and redistribute it
+   freely, subject to the following restrictions:
 
-    1. The origin of this software must not be misrepresented; you
-       must not claim that you wrote the original software. If you use
-       this software in a product, an acknowledgment in the product
-       documentation would be appreciated but is not required.
+   1. The origin of this software must not be misrepresented; you
+   must not claim that you wrote the original software. If you use
+   this software in a product, an acknowledgment in the product
+   documentation would be appreciated but is not required.
 
-    2. Altered source versions must be plainly marked as such, and
-       must not be misrepresented as being the original software.
+   2. Altered source versions must be plainly marked as such, and
+   must not be misrepresented as being the original software.
 
-    3. This notice may not be removed or altered from any source
-       distribution. */
+   3. This notice may not be removed or altered from any source
+   distribution. */
 
 /* $Header: /code/doj/alphanum.hpp,v 1.3 2008/01/28 23:06:47 doj Exp $ */
 
+
+// MODIFIED: Julien Marrec, EffiBEM, on 2018-10-26 to avoids "n_001" being recognized as the same as "n_01"
```

Comment:
Typically we haven't been adding attribution/bug issue comments like this in new E+ code and especially in third party libraries.